d8/d8batch/d8batch_demo_feed_sources.csv

```diff
@@ -0,0 +1,378 @@
+http://www.thehindu.com/?service=rss
+http://www.theguardian.com/uk/rss
+http://www.smh.com.au/rssheadlines/top.xml
+http://www.globalsecurity.org/globalsecurity-org.xml
+http://www.finanznachrichten.de/rss-news
+http://www.voanews.com/api/epiqq
+http://www.bdlive.co.za/?service=rss
+http://www.theglobeandmail.com/?service=rss
+http://feeds.bbci.co.uk/news/rss.xml#
+http://sputniknews.com/export/rss2/index.xml
```

Comment:
Why having this huge list? I _think_ a smaller number would be quite as sufficient.